rust/ql/test/library-tests/frameworks/rusqlite/main.rs

```diff
@@ -24,27 +24,27 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         )",
         (),
     )?;
-    
+
     let query = format!("INSERT INTO person (name, age) VALUES ('{}', '{}')", name, age);
 
     connection.execute(&query, ())?;    // $ sql-sink
 
     let person = connection.query_row(&query, (), |row| {    // $ sql-sink
         Ok(Person {
-            id: row.get(0)?,        // $ database-read
-            name: row.get(1)?,      // $ database-read
-            age: row.get(2)?,       // $ database-read
+            id: row.get(0)?,        // $ MISSING: database-read
+            name: row.get(1)?,      // $ MISSING: database-read
+            age: row.get(2)?,       // $ MISSING: database-read
         })
     })?;
 
     let mut stmt = connection.prepare("SELECT id, name, age FROM person")?;      // $ sql-sink
     let people = stmt.query_map([], |row| {
         Ok(Person {
```

Comment:
Just insert the following and record the failure cause as a type inferencer test.
```rust
        let row: &rusqlite::Row<'_> = row;
``